### PR TITLE
Add label for cAdvisor

### PR DESCRIPTION
**Pull request by abuccts**
Add label for cAdvisor.

### CODE REVIEW

File: rest-server/src/templates/yarnContainerScript.mustache [hunk 1]
```diff
@@ -69,6 +69,10 @@ docker run --name $docker_name \
   --security-opt apparmor:unconfined \
   --volume /var/drivers/nvidia/current:/usr/local/nvidia:ro \
   --volume /tmp/pai-root/alive/$APP_ID:/alive \
+  --label PAI_JOB_NAME={{{ jobData.jobName }}} \
+  --label PAI_USERNAME={{{ jobData.username }}} \
+  --label PAI_TASK_ROLE_NAME={{{ taskData.name }}} \
+  --label PAI_TASK_ROLE_NO={{{ idx }}} \
```
Comment (sterowang) [inline rest-server/src/templates/yarnContainerScript.mustache]: Maybe PAI_TASK_INDEX_IN_ROLE is a better name?

Comment (sterowang) [inline rest-server/src/templates/yarnContainerScript.mustache]: Please add PAI_HOSTNAME also.

File: rest-server/src/templates/yarnContainerScript.mustache [hunk 2]
```diff
@@ -69,6 +69,10 @@ docker run --name $docker_name \
   --security-opt apparmor:unconfined \
   --volume /var/drivers/nvidia/current:/usr/local/nvidia:ro \
   --volume /tmp/pai-root/alive/$APP_ID:/alive \
+  --label PAI_HOSTNAME="$(hostname)" \
+  --label PAI_JOB_NAME={{{ jobData.jobName }}} \
+  --label PAI_USERNAME={{{ jobData.username }}} \
+  --label PAI_TASK_ROLE_NAME={{{ taskData.name }}} \
   --env APP_ID=$APP_ID \
```
Comment (sterowang) [inline rest-server/src/templates/yarnContainerScript.mustache]: Please add label for "•	pai_task_index_in_role: Task index in role".

Comment (sterowang) [reply rest-server/src/templates/yarnContainerScript.mustache]: Delay to metrics implementation stage.

Comment (abuccts) [reply rest-server/src/templates/yarnContainerScript.mustache]: ok

**Pull request merged**